python/tvm/driver/tvmc/runner.py

```diff
@@ -109,21 +168,58 @@ def drive_run(args):
         Arguments from command line parser.
     """
 
-    rpc_hostname, rpc_port = common.tracker_host_port_from_cli(args.rpc_tracker)
+    path = pathlib.Path(args.PATH)
+    options = None
+    if args.device == "micro":
+        path = path / "model.tar"
```

Comment:
we can fix this in a follow-on, but i think this should be read from ServerInfo.model_library_format_path. sorry for missing that before.

Reply:
hmr, I see. Yeah I'll prepare a follow-on PR to fix it. :+1: